scripts/release-notes/index.js

```diff
@@ -279,17 +279,27 @@ const formatCommits = (commits) => {
   return lines.join('\n');
 };
 
+const formatDate = () => {
+  const now = new Date();
+  return now.toISOString().split('T')[0]; // Format as YYYY-MM-DD
```

Comment:
Nicely done!  it's not immediately clear  what the `split()` call does, so an inline comment is helpful